datalake/src/main/java/com/sequenceiq/datalake/flow/delete/event/StorageConsumptionCollectionUnschedulingRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.sequenceiq.datalake.flow.delete.event;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.datalake.flow.SdxContext;
+import com.sequenceiq.datalake.flow.SdxEvent;
+import com.sequenceiq.datalake.flow.delete.SdxDeleteHandlerSelectors;
+
+public class StorageConsumptionCollectionUnschedulingRequest extends SdxEvent {
+
+    private final boolean forced;
+
+    @JsonCreator
+    public StorageConsumptionCollectionUnschedulingRequest(
+            @JsonProperty("resourceId") Long sdxId,
+            @JsonProperty("userId") String userId,
+            @JsonProperty("forced") boolean forced) {
+        super(sdxId, userId);
+        this.forced = forced;
+    }
+
+    public static StorageConsumptionCollectionUnschedulingRequest from(SdxContext context, StackDeletionSuccessEvent payload) {
+        return new StorageConsumptionCollectionUnschedulingRequest(context.getSdxId(), context.getUserId(), payload.isForced());
+    }
+
+    @Override
+    public String selector() {
+        return SdxDeleteHandlerSelectors.UNSCHEDULE_STORAGE_CONSUMPTION_COLLECTION_EVENT.event();
```

Comment:
you can use EventSelectorUtil

Reply:
The overriding impl has been removed like above.